lib/utils/ReporterStats.js

```diff
@@ -248,6 +248,10 @@ class ReporterStats extends RunnableStats {
         // Remove the screenshot data to reduce RAM usage on the parent process
         if (type === 'screenshot') {
             runner.data = null
+        } else if (type === 'result' && runner.requestOptions.uri.path.includes('screenshot')) {
+            runner.body = null
+        } else if (type === 'aftercommand' && runner.command === 'saveDocumentScreenshot') {
```

Comment:
Also when we remove the data at this point is there any chance that reporter like `wdio-allure-reporter` are getting restricted (e.g. not being able to use that screenshots anymore)?